custom_components/bodymiscale/translations/pl.json

```diff
@@ -0,0 +1,48 @@
+{
+  "config": {
+    "error": {
+      "already_configured": "Już skonfigurowane",
+      "height_limit": "Podano za wysoki wzrost (limit: 220 cm)",
+      "invalid_date": "Błędna data"
+    },
+    "step": {
+      "options": {
+        "data": {
+          "height": "Wzrost",
+          "impedance": "Sensor impedancji",
+          "weight": "Sensor wagi"
+        },
+        "data_description": {
+          "impedance": "Jeśli waga nie podaje impedancji, pozostaw to pole puste."
+        }
+      },
+      "user": {
+        "data": {
+          "birthday": "Urodziny",
+          "gender": "Płeć",
+          "name": "Imię"
+        }
+      }
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "data": {
+          "height": "Wzrost",
+          "impedance": "Sensor impedancji",
+          "weight": "Sensor wagi"
+        },
+        "data_description": {
+          "impedance": "Jeśli waga nie podaje impedancji, pozostaw to pole puste."
+        }
+      }
+    }
+  },
+  "state": {
+    "_": {
+      "ok": "OK",
+      "problem": "Problem"
+    }
+  }
```

Comment:
```suggestion
```

Please remove this block, as it is the same in English (en.json).
As English is the fallback it will be automatically used. This save space and improves the maintainability :)

Reply:
I do change there so problem resolve...